lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -1463,6 +1470,14 @@ evaluateAndCacheCall(SILFunction &fn, SubstitutionMap substitutionMap,
       continue;
     }
 
+    if (auto *SEAI = dyn_cast<SwitchEnumAddrInst>(inst)) {
```

Comment:
You can also handle SwitchEnumInst, which is pretty much identical, but doesn't "computeLoadResult" as the first step.